src/models/__tests__/user_data_processing.test.ts

```diff
@@ -120,9 +124,7 @@ describe("createOrUpdateByNewOne", () => {
 
     const model = new UserDataProcessingModel(containerMock);
 
-    const result = await model
-      .createOrUpdateByNewOne(aUserDataProcessing)
-      .run();
+    const result = await model.upsert(aNewUserDataProcessing).run();
```

Comment:
why this change here ? it looks like [createOrUpdateByNewOne](https://github.com/pagopa/io-functions-commons/blob/47a80c858b77fef036b27efd209c66b952e67440/src/models/user_data_processing.ts#L110) does more than a simple upsert